hw4/index.js

```diff
@@ -0,0 +1,29 @@
+function Person (name, age)	{
+    this.name = name;
+    this.age = age;
+}
+
+Person.prototype.introduce = function(){
+    return 'My name is ' + this.name + ' and I am ' + this.age;
+};
+
+var	matvey = new Person('Matvey', 30);
+var	jack = new Person('Jack', 40);
+
+
+console.log( matvey.introduce() ); //	My name is Matvey and I am 30
+console.log( jack.introduce() ); //	My name is Jack and I am 40
+
+console.log( "----------------------------" );
+
+function myNew (className, ...args) {
+    let myPerson = new className(args[0], args[1]);
```

Comment:
@matveyprupas Here you don't really use all the parameters passed, just first 2 of them.

The task requires another thing:
"This	function	should	take	one	function	parameter	(the	constructor),	plus	an	unknown	
number	of	additional	parameters	of	any	type	(arguments	for	the	constructor)."

Reply:
Agree. Fixed